docs/admin/components/list.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: List components
```

Comment:
I don't love this structure: I don't see the connection between `SelectField` and `Repeater`.